roles/kubernetes/control-plane/templates/kubeadm-config.v1beta4.yaml.j2

```diff
@@ -0,0 +1,545 @@
+apiVersion: kubeadm.k8s.io/v1beta4
```

Comment:
Does it need to be the same, like:

```
apiVersion: kubeadm.k8s.io/{{ kubeadm_config_api_version }}
```

Reply:
It doesn't need. Because `roles/kubernetes/control-plane/tasks/kubeadm-setup.yml` like this

```
- name: Kubeadm | Create kubeadm config
  template:
    src: "kubeadm-config.{{ kubeadm_config_api_version }}.yaml.j2"
```

The reason for this design is that the v1beta4 of this file has more changes, so it is separated into another file, there is no need to write this variable.